backend/docs/buildingsV1.yml

```diff
@@ -118,16 +132,17 @@ components:
       properties:
         name:
           type: string
-        level:
+        floor_level:
           type: integer
+          
     Floor:
       type: object
       properties:
         name:
           type: string
-        level:
+        floor_level:
```

Comment:
Are we not already tying a floor's level to the object by providing the "level" property in the floor object? Is naming this attribute `floor_level` redundant?